tools/update-pa-firewall.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python
```

Comment:
Is there a reason you wish to add this file? I am not doubting that it may be useful to you but will it be useful to others in their own environment? I don't use palo alto firewalls so I am not exactly sure. There is also the fact that by accepting this file we are on-boarding the responsibilities around the file. We could say that things under the tools directory are unsupported but I would rather not do that either.

Let me know the reasoning behind this addition.